README.md

```diff
@@ -76,6 +76,7 @@ The devices below are fully feature supported by OpenRazer, which means all avai
 | Razer Book 13 (2020)                          |  1532:026A  |
 | Razer Blade 15 Advanced (Early 2021)          |  1532:026D  |
 | Razer Blade 15 Advanced (Mid 2021)            |  1532:0276  |
+| Razer Blade 14 (2021)                         |  1532:2070  |
```

Comment:
Should be `1532:0270` and move it one line up to keep it sorted

Reply:
@z3ntu  @BeanieBen9990  Made a pull request for this change.

Reply:
@z3ntu the formatting as suggested has been merged :smile: